### PR TITLE
Fix/pr63

### PR DESCRIPTION
**Pull request by petermisik**

### CODE REVIEW

**Review by bmorris-revacomm — APPROVED** [review]
Looks good to me!

File: Logging/Views/Overview.storyboard [hunk 1]
```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="17506" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="BXu-yv-Oz6">
+<<<<<<< HEAD
+    <device id="ipad9_7" orientation="portrait" layout="fullscreen" appearance="light"/>
+=======
     <device id="ipad12_9rounded" orientation="portrait" layout="fullscreen" appearance="light"/>
+>>>>>>> origin/main
```
Comment (Pete-Hoch) [inline Logging/Views/Overview.storyboard:7]: This should be resolved before pushing.

File: Logging/Views/Overview.storyboard [hunk 2]
```diff
@@ -73,7 +77,11 @@
                                 <nil key="highlightedColor"/>
                             </label>
                             <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Your forms for this mission:" textAlignment="center" lineBreakMode="tailTruncation" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="8wZ-NV-sMB" customClass="PBLLabelLight" customModule="Logging" customModuleProvider="target">
+<<<<<<< HEAD
+                                <rect key="frame" x="148.5" y="379" width="471.5" height="48"/>
+=======
                                 <rect key="frame" x="276.5" y="476" width="471.5" height="48"/>
+>>>>>>> origin/main
```
Comment (Pete-Hoch) [inline Logging/Views/Overview.storyboard:84]: Same here.

**Review by Pete-Hoch — CHANGES_REQUESTED** [review]
Some conflict shrapnel in the Overview.storyboard.

**Comment by Pete-Hoch** [comment]
And you can rebase off of release/0.1.0 now.

**Comment by cbrech** [comment]
Once you rebase and push new code into this PR it will re-trigger the Test workflow and on pass you will be able to merge

**Comment by petermisik** [comment]
Changes made.

**Review by Pete-Hoch — APPROVED** [review]
Changes look good. Just clear the conflict and squash & merge!

**Pull request merged**